lib/publiccloud/azure.pm

```diff
@@ -16,26 +16,16 @@ use testapi qw(is_serial_terminal :DEFAULT);
 use utils qw(script_output_retry);
 use publiccloud::azure_client;
 
-has tenantid => sub { get_var('PUBLIC_CLOUD_AZURE_TENANT_ID') };
-has subscription => sub { get_var('PUBLIC_CLOUD_AZURE_SUBSCRIPTION_ID') };
-has region => sub { get_var('PUBLIC_CLOUD_REGION', 'westeurope') };
-has username => sub { get_var('PUBLIC_CLOUD_USER', 'azureuser') };
 has resource_group => 'openqa-upload';
 has storage_account => 'openqa';
 has container => 'sle-images';
 has lease_id => undef;
 has vault => undef;
-has provider_client => undef;
```

Comment:
is it correct ? we still using it , right ?

Reply:
ok , you moved it to provider . nice !

Reply:
yes :)